cmd/clusterctl/cmd/util.go

```diff
@@ -38,15 +38,14 @@ func printYamlOutput(printer client.YamlPrinter, outputFile string) error {
 	if err != nil {
 		return err
 	}
-	outputFile = filepath.Clean(strings.TrimSpace(outputFile))
```

Comment:
For the `if outputFile == ""` check below, I think it may be safer to keep the `strings.TrimSpace(outputFile)` here, then only do the `filePath.Clean()` below after the check.

Reply:
done